RxTodoTests/Sources/Tests/TaskEditViewModelTests.swift

```diff
@@ -33,12 +34,12 @@ class TaskEditViewModelTests: XCTestCase {
   func testTitle() {
     RxExpect { test in
       let viewModel = TaskEditViewModel(mode: .new)
-      test.assertNextEqual(viewModel.title.asDriver(), [""])
+        test.assertNextEqual(viewModel.title.asDriver().filterNil().map {$0}, [""])
```

Comment:
I misunderstood the comment 😭  

what do you prefer `.filterNil().map` or `.filterNil()` both way passed unittest